examples/typescript/example.ts

```diff
@@ -35,8 +36,60 @@ vm.attachChangeEvent(() => {
     console.log("Something changed!");
 });
 
+// VB-Audio Callback example:
+
+// Index at which the first bus output starts in inputChannels
+let inputOffset = 0;
+switch (vm.$type) {
+    case "voicemeeter": {
+        inputOffset = 12;
+        break;
+    }
+    case "voicemeeterBanana": {
+        inputOffset = 22;
+        break;
+    }
+    default: {
+        inputOffset = 34;
+        break;
+    }
+}

```

Comment:
use better an object here:

```ts
const inputOffsets = {
"voicemeeter": 12,
"voicemeeterBanana": 22,
}
const inputOffset = inputOffsets[vm.$type] ?? 34;
```

Should we add these offsets to the library? as "helper" constants?